sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -159,6 +159,8 @@
 mi = mile = miles = Quantity("mile")
 mi.set_global_relative_scale_factor(5280, feet)
 
+angstrom = angstroms = Quantity("angstrom", latex_repr=r'\r{A}')
```

Comment:
I would recommend to add `.set_global_relative_scale_factor(... , meter)` here because the scale factor relative to meters is the same in all unit systems.